app/javascript/lib/vue-components/modules/BeesWarmChart.vue

```diff
@@ -0,0 +1,353 @@
+<template>
+  <div class="beeswarm-container">
+    <div class="beeswarm-tooltip"></div>
+    <svg class="beeswarm-plot" :width="svgWidth" :height="svgHeight"></svg>
```

Comment:
This is the third time I see something alike. The Vue-d3 combination is more powered when the markup is drawn by Vue, instead.
Like this line (where you're calculating the width/height in vue), you should do the same for the rest of the code (`<g>`, `<line v-for...>`) In this way, you'll save the setup/build functions. Besides, you're be able to add Tooltips as components, instead of hard-coded